config/githooks/pre-push

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+cd $(git rev-parse --show-toplevel)
+
+# copyright check
+copyright=true
+if git grep -l "^\(#\|//\).*\(\-2020\|\ 2010,\)" > matches.txt
```

Comment:
Can we avoid the code duplication between this file and `.github/workflows/repo-check.yml`?

Reply:
I thought about this too. I think we could, yes.